worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageLockWrapper.java

```diff
@@ -25,16 +22,14 @@
 public class CoverageLockWrapper {
 
     private static final String COVERAGE_LOCK_NAME = "COVERAGE_LOCK";
-    private static final int LOCK_TIME = 60_000; // 60 seconds
+    private static final int TEN_MINUTES = 600_000; // 6 minutes in milliseconds
```

Comment:
The '6 minutes' part of the comment is confusing.  How about ditching the comment entirely and just name the constant

TEN_MINUTES_IN_MSEC ?